includes/Channel.hpp

```diff
@@ -13,40 +13,31 @@
 #ifndef INCLUDES_CHANNEL_HPP_
 #define INCLUDES_CHANNEL_HPP_
 
-#include <arpa/inet.h>
-#include <fcntl.h>
-#include <netinet/in.h>
-#include <poll.h>
-#include <sys/socket.h>
-#include <sys/types.h>
-#include <unistd.h>
-
-#include <csignal>
-#include <cstdlib>
-#include <cstring>
-#include <iostream>
-#include <vector>
 #include <map>
-#include <algorithm>
+#include <string>
 
-#include "../includes/Client.hpp"
+#include "Client.hpp"
 
-class Channel {
- private:
-  std::string                _name;
-  std::string                _topic;
-  std::map<int, Client>      _clients;
+class Client;
```

Comment:
We don't need class Client here because Client class doesn't include Channel class.